### PR TITLE
Simplified formatting of the notebooks to ensure correct rendering of documentation

### PR DESCRIPTION
**Pull request by apaleyes**
Fancy HTML features completely messed up Sphinx documentation rendering, so I've decided to change all notebooks to markdown-only format.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/notebooks/Emukit-how-to-write-a-notebook.ipynb [hunk 1]
```diff
@@ -72,36 +58,30 @@
...
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Specific imports that are used in a section should be loaded at the beggining of that section. It is ok, if an \n",
-    "# import is repeated multiple times over the notebook\n",
+    "# Specific imports that are used in a section should be loaded at the beggining of that section.\n",
```
Comment (mmahsereci) [inline examples/notebooks/Emukit-how-to-write-a-notebook.ipynb]: typo: be\_g\_inning

File: examples/notebooks/Emukit-how-to-write-a-notebook.ipynb [hunk 2]
```diff
@@ -4,28 +4,14 @@
...
-    "        Title of notebook ... with <b style=\"color:#FF9900\">Emukit</b>\n",
-    "    </h1>\n",
-    "    <br>\n",
-    "    <hr>\n",
-    "    <br>\n",
-    "<center>\n",
-    "    By *Author 1* and *Author 2* with edits by *Editor 1*\n",
-    "    \n",
-    "    Place, date"
+    "# Title of notebook (... with Emukit)"
```
Comment (mmahsereci) [inline examples/notebooks/Emukit-how-to-write-a-notebook.ipynb]: why brackets around "... with Emukit?"

Comment (apaleyes) [reply examples/notebooks/Emukit-how-to-write-a-notebook.ipynb]: To indicate this is optional. Maybe I should remove " (... with Emukit)" altogether?

File: examples/notebooks/Emukit-how-to-write-a-notebook.ipynb (hunk 1 above)
Comment (apaleyes) [reply examples/notebooks/Emukit-how-to-write-a-notebook.ipynb]: fixed, thanks

**Pull request merged**